### ISSUE TITLE
Wrong install directory

### ISSUE THREAD

**Issue by iluetkeb** [cbandera/rosparam_handler#49]
utilities.hpp is currently installed to <install-prefix>/include/rosparam_handler/rosparam_handler/utilities.hpp -- note the double occurence of "rosparam_handler" in the path. This leads to failures when including it in generated parameter configuration files.

This problem does not occur when you do not have an "install" directory in your workspace, which may explain why it is commonly missed. However, it is unfortunately also present in the currently distributed version of the rosparam_handler package!

I surmise the cause of the problem is that because dynamic_reconfigure already creates the target directory, and thus the subsequence copy-step accidentally copies the whole directory instead of its contents.

A quick fix is to replace the "DIRECTORY" form of the install directive with the "FILES" form.  See the following patch. Since you only have one header, it should be fine.

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index a0d0437..d10ecfb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,7 @@ install(
 )
 
 install(
-        DIRECTORY include/rosparam_handler/
+        FILES include/rosparam_handler/utilities.hpp
         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
 )

```

**Comment by artivis** [comment]
Hi, thanks for reporting the issue. It is already fixed in PR #48 that is about to be merged :+1:

**Comment by artivis** [comment]
FYI, #48 has been merged, I'm waiting for #50 to be merged as well before releasing 0.1.3 which will fix this issue.

**Comment by artivis** [comment]
Release 0.1.3 containing the installation fix #48 is under release process. In a week roughly it should be available from the ros repo.
